course_contents/2_intro_to_python/notes/01. Conditions.md

```diff
@@ -0,0 +1,101 @@
+### The `if` statement
+- the `if` statement is used to check if a conditional is `True`
+- syntax for writing the `if` statement:
+    1. use the `if` keyword at the beginning of the statement
+    2. follow the keyword with a correctly written conditional
+    3. finish the expression with colon 
+    4. block of code that will be executed if the statement passes - should be indented (usually by 4 spaces)
+
+An example of proper writing of the `if` statement:
+
+```py
+friend = "Rolf"
+name = input("Enter your name: ")
+if name == friend:
+    print(f"{name} is a friend")
+```
+
+Two examples of incorrect writing of the `if` statement:
+
+```py
+if name = friend:
+    print(f"{name} is a friend")
+```
+```py
+if name == friend:
+print(f"{name} is a friend")
+```
+**Extra resources**
+- [What is an if statement?](https://python.tecladocode.com/2_countries_visited/2_if_statements.html#what-is-an-if-statement)
+
+### The compound `if` statement - `elif` keyword
+- `elif` keyword is used for chaining multiple if statements
+- `elif` block will be entered only if the previous `if`/`elif` block doesn't get entered
+- the syntax is the same as with writing the `if` statement, the difference being that the `elif` keyword is used instead of `if`
```

Comment:
This is quite long. Could we simplify it a little?

Reply:
As I mentioned above, I tried to simplify stuff as much as I could, so if any of you guys have any suggestions feel free to send them my way! 👍 
I will try to shorten everything when I go through the notes again as well